### PR TITLE
Skills demographics visualization (admin)

### PR DESCRIPTION
**Pull request by SanaBob**
+ skills endpoint
+ skillsGender react component (word cloud + d3 histogram)
+ skillsYearLevel react component (word cloud + d3 histogram)

### CODE REVIEW

**Review by liewshanhong — CHANGES_REQUESTED** [review]
chart overlaps with legend

File: skills_DS/skills_DS/settings.py [hunk 1]
```diff
@@ -163,7 +163,7 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 EMAIL_HOST_USER = 'cosc448atesting@gmail.com'
-EMAIL_HOST_PASSWORD = ''
+EMAIL_HOST_PASSWORD = 'xEpdu0-waqqux-repwuf'
```
Comment (brianzhouzc) [inline skills_DS/skills_DS/settings.py]: Time to change the password again xD

I just notice we never merged the change for the environment variable change. I will open a pull request on it later tomorrow.

**Review by brianzhouzc — APPROVED** [review]
I will suggest creating a component for SkillsGender.js and SkillsYear.js since the majority of the code is the same, and can be reused and lower maintenance costs. 

However, I understand that it might be difficult to change it at this stage, so I will leave it to you.

File: skills_DS/skills_DS/settings.py (hunk 1 above)
Comment (SanaBob) [reply skills_DS/skills_DS/settings.py]: I'll hide it again lol

**Review by liewshanhong — APPROVED** [review]

**Pull request merged**